pkg/platform/runtime/buildexpression/testdata/buildexpression.json

```diff
@@ -2,7 +2,7 @@
   "let": {
     "runtime": {
       "solve_legacy": {
-        "at_time": "2023-04-27T17:30:05.999000Z",
+        "at_time": "$at_time",
```

Comment:
I'm not changing all test cases because I think this field can be either `$at_time` or a valid timestamp, and we want to be okay with both cases.